moto/batch/responses.py

```diff
@@ -215,8 +217,9 @@ def describejobs(self) -> str:
     def listjobs(self) -> str:
         job_queue = self._get_param("jobQueue")
         job_status = self._get_param("jobStatus")
+        filters = self._get_param("filters")
```

Comment:
We should have some tests here as well, to verify this parameter is handled correctly

Reply:
done